scripts/quests/hiddenQuests/New_Character_Cutscenes.lua

```diff
@@ -0,0 +1,341 @@
+                    npcUtil.giveItem(player, xi.items.ADVENTURERS_COUPON)
+
+                    -- The HP position is slightly different than the CS exit.
+                    player:setPos(-40.0, -5, 100, 64)
+                    player:setHomePoint()
+                    player:setPos(-40.611, -5, 102.5, 57) -- Move back to CS exit position
+
+                    quest:setVar(player, 'notSeen', 0)
+                end,
+            },
+        },
+
+        [xi.zone.WINDURST_WOODS] =
+        {
+            onZoneIn =
+            {
+                function(player, prevZone)
+                    local cutsceneFlags = 0
+
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.UNKNOWN_1)
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.NO_PCS)
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.NO_NPCS)
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.UNKNOWN_2)
+
+                    return { 367, -1, cutsceneFlags }
+                end
+            },
+
+            onEventFinish =
+            {
+                [367] = function(player, csid, option, npc)
+
+                    local ID = zones[player:getZoneID()]
+                    player:messageText(player, ID.text.MAP_MARKER_TUTORIAL)
+                    -- If you don't get the coupon, tough luck. Retail doesn't give you a chance to get it again.
+                    npcUtil.giveItem(player, xi.items.ADVENTURERS_COUPON)
+
+                    player:setPos(30, 2, -40, 128)
+                    player:setHomePoint()
+
+                    quest:setVar(player, 'notSeen', 0)
+                end,
+            },
+        },
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            onZoneIn =
+            {
+                function(player, prevZone)
+                    local cutsceneFlags = 0
+
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.UNKNOWN_1)
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.NO_PCS)
+                    cutsceneFlags = bit.bor(cutsceneFlags, xi.cutscenes.params.UNKNOWN_2)
+
+                    return { 305, -1, xi.cutscenes.params.NO_OTHER_ENTITY }
+                end
+            },
+
+            onEventFinish =
+            {
+                [305] = function(player, csid, option, npc)
+
+                    local ID = zones[player:getZoneID()]
+                    player:messageText(player, ID.text.MAP_MARKER_TUTORIAL)
+                    -- If you don't get the coupon, tough luck. Retail doesn't give you a chance to get it again.
+                    npcUtil.giveItem(player, xi.items.ADVENTURERS_COUPON)
+
+                    player:setPos(-120, -5.5, 175, 48)
+                    player:setHomePoint()
+
+                    player:setPos(-140, -7, 172, 32)
+                    quest:setVar(player, 'notSeen', 0)
```

Comment:
Could use quest complete here it will clear all the vars then move the item to rewards section and remove the nocUtlgiveItem to all the finishes